contracts/sponsorship/BiconomySponsorshipPaymaster.sol

```diff
@@ -4,20 +4,21 @@ pragma solidity ^0.8.26;
 /* solhint-disable reason-string */
 
 import "../base/BasePaymaster.sol";
-import "account-abstraction/contracts/core/UserOperationLib.sol";
-import "account-abstraction/contracts/core/Helpers.sol";
-import { SignatureCheckerLib } from "solady/src/utils/SignatureCheckerLib.sol";
-import { ECDSA as ECDSA_solady } from "solady/src/utils/ECDSA.sol";
-import { BiconomySponsorshipPaymasterErrors } from "../common/Errors.sol";
-import { ReentrancyGuard } from "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
-import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-import { SafeTransferLib } from "solady/src/utils/SafeTransferLib.sol";
+import "@account-abstraction/contracts/core/UserOperationLib.sol";
+import "@account-abstraction/contracts/core/Helpers.sol";
+import { SignatureCheckerLib } from "@solady/src/utils/SignatureCheckerLib.sol";
+import { ECDSA as ECDSA_solady } from "@solady/src/utils/ECDSA.sol";
+import { BiconomySponsorshipPaymasterErrors } from "../common/BiconomySponsorshipPaymasterErrors.sol";
+import { ReentrancyGuardTransient } from "@openzeppelin/contracts/utils/ReentrancyGuardTransient.sol";
```

Comment:
curious what is the benefit of this ReentrancyGuardTransient
over ReentrancyGuard

Reply:
Gas savings. Uses tload and tstore opcodes vs sload and sstore.